test/framework/kubetest/run.go

```diff
@@ -143,6 +144,9 @@ func Run(ctx context.Context, input RunInput) error {
 		tmpKubeConfigPath: "/tmp/kubeconfig",
 		reportDir:         "/output",
 	}
+	for k, v := range input.ExtraMounts {
+		volumeMounts[k] = v
+	}
 	user, err := user.Current()
 	if err != nil {
 		return errors.Wrap(err, "unable to determine current user")
```

Comment:
@randomvariable Does it make a difference that before it was part of `e2eArgs` and now of `config` (and there's another `--` in between) (line 171/172)

Reply:
seemingly not.

Reply:
Is that based on tests in CAPA or CAPI? I would run our test which uses this part, but I have to fix this test first.

Reply:
Fiddling around in CAPA at the moment.